server/game/cards/01_SOR/units/RuthlessRaider.ts

```diff
@@ -0,0 +1,36 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { CardType, RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class RuthlessRaider extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '1047592361',
+            internalName: 'ruthless-raider'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'Deal 2 damage to an enemy base and 2 damage to an enemy unit',
+            when: {
+                onCardPlayed: (event, context) => event.card === context.source,
+                onCardDefeated: (event, context) => event.card === context.source
+            },
+            targetResolvers: {
```

Comment:
This is kind of getting into the weeds a little and into a place where the card implementation tools need some improvement, but let's remove the `targetResolvers` property and replace it with an `immediateEffect` property that does this:

```typescript
  immediateEffect: AbilityHelper.immediateEffects.simultaneous([
      AbilityHelper.immediateEffects.selectCard({
          cardTypeFilter: WildcardCardType.Unit,
          controller: RelativePlayer.Opponent,
          innerSystem: AbilityHelper.immediateEffects.damage({ amount: 2 }),
      }),
      AbilityHelper.immediateEffects.damage((context) => ({ amount: 2, target: context.source.controller.opponent.base }))
  ])
```

The reason is that `theirBase` shouldn't really be a target resolver since target resolvers are for when players will have to choose a target, but that isn't true for `theirBase` since it will always be the opponent's base specifically.

So we need to mix a target resolver (`theirUnit`) with a system that doesn't need a target resolver and that's kind of ugly to do right now, as you see above. Basically the `selectCard()` call acts like a target resolver in place `theirUnit` and the `theirBase` handles the base damage by selecting that card explicitly.

Feel free to follow up in discord if you have questions